build.sbt

```diff
@@ -77,6 +76,28 @@ lazy val zioSchemaJS = zioSchema.js
 
 lazy val zioSchemaJVM = zioSchema.jvm
 
+lazy val zioSchemaJson = crossProject(JSPlatform, JVMPlatform)
```

Comment:
We should aggregate `zioSchemaJsonJS` and `zioSchemaJsonJVM` under `root`. Right now `JsonCodecSpec` is not running in the CI job.